sample_apps/simple_app/instructions.md

```diff
@@ -0,0 +1,468 @@
+# Simple App - Instructions 
+
+This document gives detailed instructions for building and running the sample application and generating the policy for the Certifier Service using the policy utilities.
+
+This simple_app sample program but provides an example of initializing
+and provisioning the Certifier Service with utility generated keys, measurements and policy.
+
+The sample program will still need to construct the statement "The attestation-key says the
+enclave-key speaks-for the program".  This is the attestation.
+
+Except for the ancillary files attest_key_file.bin, example_app.measurement and
+platform_attest_endorsement.bin which are needed because of the simulated-enclave, this
+example closely models needed for a real (but simple) deployment. In addition, this example
+embeds the policy key in the application using embed_policy_key.exe.
+
+Read the [policy_key_notes.txt](policy_key_notes.txt) in the simple_app directory and
+[policy_utilities_info.txt](../../utilities/policy_utilities_info.txt) as  a background.
+
+
+$CERTIFIER_PROTOTYPE is the top level directory for the Certifier repository. 
+It is helpful to have a shell variable for it, e.g., :
+
+```shell
+$ export CERTIFIER_PROTOTYPE=~/src/github.com/jlmucb/crypto/v2/certifier-framework-for-confidential-computing
+```
+
+$EXAMPLE\_DIR is this directory containing the example application.  Again, a shell variable
+is useful.
+
+```shell
+$ export EXAMPLE_DIR=$CERTIFIER_PROTOTYPE/sample_apps/simple_app
+```
+
+----
+## Step 1: Build the utilities
+
+```shell
+$ cd $CERTIFIER_PROTOTYPE
+$ cd utilities
+$ make -f cert_utility.mak
+$ make -f policy_utilities.mak
+```
+
+## Step 2:  Create a directory for the provisioning files
+```shell
+$ mkdir $EXAMPLE_DIR/provisioning
+```
+
+
+## Step 3: Generate the policy key and self-signed cert
+
+```shell
+$ cd $EXAMPLE_DIR/provisioning
+
+$ $CERTIFIER_PROTOTYPE/utilities/cert_utility.exe        \
+      --operation=generate-policy-key-and-test-keys      \
+      --policy_key_output_file=policy_key_file.bin       \
```

Comment:
I specifically preferred this style of listing each `--<option>` one on each line, so that each option stands out on its own.

I found that in the older format (in instructions.txt) with multiple `--options` on one line, it gets difficult to see what the invocation for each command actually is doing.